backend/app/assets/javascripts/spree/backend/format_money.js

```diff
@@ -0,0 +1,10 @@
+(function() {
+  var defaultLocale = document.documentElement.lang;
+
+  Spree.formatMoney = function(amount, currency, options) {
```

Comment:
I can appreciate that, but I'm not 100% sure that this will be the permanent implementation of this method. We might have to end up including `accounting.js` or similar, which we would not want cluttering up the frontend. For that reason, I want to keep this backend-only.